src/rules/binding-positions.ts

```diff
@@ -42,7 +42,6 @@ const rule: Rule.RuleModule = {
     ): boolean {
       return (
         expr !== undefined &&
-        expr !== null &&
```

Comment:
maybe we still want this one? if it can be null? though it built so maybe we don't?

Reply:
It doesn't show that it can be `null`. Typing above shows `ESTree.TemplateElement | undefined` and the intelli-sense doesn't yell at me

Reply:
ah yeah its our own function, fair enough. its right :D